packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -596,6 +596,23 @@ class GoTrueClient {
     }
   }
 
+  /// Gets the current user details from current session or custom [jwt]
+  Future<UserResponse> getUser(String? jwt) async {
```

Comment:
We probably want to make the parameter optional I think.
```suggestion
  Future<UserResponse> getUser([String? jwt]) async {
```